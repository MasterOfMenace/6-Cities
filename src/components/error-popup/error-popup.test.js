import React from 'react';
import renderer from 'react-test-renderer';
import ErrorPopup from './error-popup.jsx';

const mockMessage = `Message`;

it(`Правильное отображение компонента ErrorPopup`, () => {
  const tree = renderer.create(
      <ErrorPopup message={mockMessage} onButtonClick={()=>{}}/>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
