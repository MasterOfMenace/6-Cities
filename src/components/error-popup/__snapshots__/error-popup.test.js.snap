// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Правильное отображение компонента ErrorPopup 1`] = `
<div
  className="popup popup-error"
  style={
    Object {
      "backgroundColor": "rgba(0,0,0, 0.5)",
      "height": "100%",
      "left": 0,
      "position": "fixed",
      "top": 0,
      "width": "100%",
      "zIndex": 9999,
    }
  }
>
  <div
    className="popup popup__inner"
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgb(255, 255, 255)",
        "display": "flex",
        "flexDirection": "column",
        "font": "inherit",
        "fontSize": "18px",
        "left": "calc(80px)",
        "padding": "40px",
        "position": "absolute",
        "top": "40%",
        "width": "calc(100% - 160px)",
      }
    }
  >
    <p
      className="popup__text"
    >
      Message
    </p>
    <button
      className="popup__close-button button"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#4481c3",
          "borderRadius": "3px",
          "color": "#fff",
          "padding": "16px 30px 13px",
        }
      }
    >
      Close
    </button>
  </div>
</div>
`;
